packpack

```diff
@@ -88,6 +88,7 @@ usage() {
     echo " * DIST - the version of distribution, e.g. xenial or 24 (${DIST})"
     echo " * BUILDDIR - directory used for out-of-source build (${BUILDDIR})"
     echo " * DOCKER_REPO - Docker repository to use (${DOCKER_REPO})"
+    echo " * USE_LOCAL_IMAGE - if not empty try to use local docker image first"
```

Comment:
It would be nice to add the new option into the README ([configuration](https://github.com/packpack/packpack?tab=readme-ov-file#configuration) section) as well.

Reply:
Done.